c2corg_ui/static/js/map/diffmap.js

```diff
@@ -81,10 +80,13 @@ app.DiffMapController = function(mapFeatureCollection) {
   app.utils.setupSmartScroll(mouseWheelZoomInteraction);
 
   if (!mapFeatureCollection) {
-    this.map.setView(new ol.View({
-      center: app.MapController.DEFAULT_CENTER,
-      zoom: app.MapController.DEFAULT_ZOOM
-    }));
+    var mapSize = this.map.getSize();
+    if (mapSize) {
+      this.map.getView().fit(app.MapController.DEFAULT_EXTENT, mapSize,
+      /** @type {olx.view.FitOptions} */ ({
```

Comment:
Honestly I don't want to spend more time on this PR. We can't spend days on such a small change ("default map extent"). OK for merging the PR?